lib/processes/visTestPattern.hpp

```diff
@@ -10,17 +11,32 @@
 #include "Stage.hpp"
 #include "buffer.h"
 #include "bufferContainer.hpp"
+#include "datasetManager.hpp"
+#include "restServer.hpp"
 #include "visUtil.hpp"
 
+#include "json.hpp"
+
 #include <fstream>
 #include <stddef.h>
 #include <string>
 #include <vector>
 
+using json = nlohmann::json;
+
+#define REPORT_PRECISION 7
```

Comment:
Should this be made a function of the tolerance?

Reply:
That's a good idea.